### PR TITLE
fix operator of throttle

### PR DESCRIPTION
**Pull request by matthewyan**

### CODE REVIEW

File: RxMarbles/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -42,7 +42,7 @@
 "concat" = "Emit the emissions from two or more Observables without interleaving them.";
 
 //throttle
-"throttle" = "Only emit an item from an Observable if a particular timespan has passed without it emitting another item.";
+"throttle" = "Emit the first and the latest item emitted by the source Observable during sequential time windows of a specified duration.";
```
Comment (Vkt0r) [inline RxMarbles/en.lproj/Localizable.strings:45]: I think it wouldn't hurt to translate this to Russian using Google Translate and update the `ru.lproj` too isn't? @gre4ixin Can help ensure the translation is correct 🙏.

**Comment by Vkt0r** [comment]
@matthewyan Thanks for taking the time the time to create this PR. Looks very good, awesome job. Just let's update the translation and let's keep it open a couple of days in case @yury want to take a look too.

**Comment by matthewyan** [comment]
@Vkt0r thanks, I have modified the Russian translation

**Review by gre4ixin — COMMENTED** [review]
it would be more correct to translate "Испускает первый и последний элемент Observable в течение последовательного промежутка времени"

**Comment by gre4ixin** [comment]
@matthewyan I corrected the translation a bit, will you fix it in the commit or should I do a separate PR?)

**Comment by matthewyan** [comment]
@gre4ixin Thanks, I have submitted this change

**Comment by yury** [comment]
Thank you!

**Pull request merged**